src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    subtract(value) {
+      this.result -= value;
+
+      return this.result;
```

Comment:
The method `subtract` correctly uses the arithmetic operator with assignment operator `-=`, adhering to the [CODE STYLE] rule.